static/sass/_pattern_careers-progression.scss

```diff
@@ -0,0 +1,379 @@
+@import "../../node_modules/vanilla-framework/scss/patterns_icons";
+
+@mixin canonical-p-careers-progression {
+  body {
+    // Fix overflown video
+    overflow-x: hidden;
+  }
+
+  .p-strip--careers-progression {
+    @extend %vf-strip;
+
+    background-color: #261f24;
+    color: vf-determine-text-color(#261f24);
+
+    &.is-hero {
+      padding-top: 16.5rem;
+      // Fix overflown video
+      overflow-x: hidden;
+
+      @media screen and (min-width: $breakpoint-small) and (max-width: 1112px) {
+        padding-top: 2.5rem;
+      }
+
+      @media screen and (max-width: ($breakpoint-small - 1)) {
+        padding-top: $sp-large;
+      }
+    }
+
+    .p-roles {
+      border-right: 1px solid rgb(255 255 255 / .25);
+      display: flex;
+      flex-direction: column;
+      height: 100%;
+      justify-content: space-between;
+      padding-right: 1.8rem;
+
+      :last-child {
+        border-right: 0;
+      }
+
+      @media screen and (min-width: $breakpoint-small) and (max-width: $breakpoint-large) {
+        margin-top: 1rem;
+      }
+
+      @media screen and (max-width: $breakpoint-small) {
+        border: 0;
+      }
+
+      img {
+        border-radius: 0;
+      }
+    }
+
+    .p-strip--hero-title {
+      font-size: 6.625rem;
+      line-height: 6rem;
+      margin-bottom: 6rem;
+
+      span {
+        margin-top: 3rem;
+        @media screen and (max-width: ($breakpoint-small - 1)) {
+          margin-top: 1.5rem;
+        }
+      }
+
+      @media screen and (min-width: $breakpoint-small) and (max-width: $breakpoint-large) {
+        font-size: 3.625rem;
+        line-height: 3rem;
+      }
+
+      @media screen and (max-width: ($breakpoint-small - 1)) {
+        font-size: 2rem;
+        line-height: 1.5rem;
+        margin-bottom: 1.5rem;
+      }
+    }
+  }
+
+  @keyframes slide-right {
+    from {
+      opacity: 0;
+      transform: translateY(20%);
+      width: 0;
+    }
+
+    to {
+      opacity: 1;
+      overflow: visible;
+      transform: translateY(0%);
+      width: var(--width);
+    }
+  }
+
+  .u-animation--slide-right {
+    --width: 100%;
+
+    &-separator {
+      --width: 110px;
+    }
+
+    animation: slide-right map-get($animation-duration, sleepy) 1 forwards
+      cubic-bezier(0.6, 0.12, 0.28, 0.75);
+    display: block;
+    opacity: 0;
+    white-space: nowrap;
+    overflow: hidden;
+
+    @media screen and (max-width: ($breakpoint-small - 1)) {
+      margin: $sp-large 0;
+    }
+  }
+
+  .p-separator {
```

Comment:
Won't this animate all the separators across the site?

Reply:
I couldn't achieve the same effect with the Vanilla styles, so I renamed it as a custom class